README.md

```diff
@@ -70,12 +74,6 @@ Thanks go out to people who offered advice or otherwise made this project possib
 ## Donations
 If you would like to show your appreciation or even help fund development, the project has a [Patreon](https://www.patreon.com/Vita3K) page.
 
-## Supporters
```

Comment:
I think petmac added this as a part of the "Nibble" and above patreon rewards

Reply:
Fixed.